index.bs

```diff
@@ -1322,13 +1322,13 @@ sends only the result to the authenticator. The authenticator signs over the com
 
 The goals of this design can be summarized as follows.
 
-- The scheme for generating signatures should accommodate cases where the link between the client platform and authenticator
+- The scheme for generating signatures SHOULD accommodate cases where the link between the client platform and authenticator
     is very limited, in bandwidth and/or latency. Examples include Bluetooth Low Energy and Near-Field Communication.
 
-- The data processed by the authenticator should be small and easy to interpret in low-level code. In particular, authenticators
-    should not have to parse high-level encodings such as JSON.
+- The data processed by the authenticator SHOULD be small and easy to interpret in low-level code. In particular, authenticators
+    SHOULD NOT have to parse high-level encodings such as JSON.
```

Comment:
This is descriptive - not normative language. Please back both of the changes in this sentence out.

Reply:
done.